### PR TITLE
Use VAOS v2 endpoints on facility choice page

### PR DESCRIPTION
**Pull request by jbalboni**
## Description

Note: Most of the added lines here are mock data, not code.

This PR: 
- Updates our call to get facilities and VATS settings to use VAOS service and MFS v2 endpoints via the v2 vets-api endpoints
- Removes some unused code around facilities and organizations
- Unifies some mocks and helpers related to parent facilities with regular facility helpers
- Adds some mock data from staging for facilities and scheduling configurations
- Fixes some snake casing in existing mock code

## Testing done
Local and unit testing

## Acceptance criteria
- [ ] Build passes and facility list is correct

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/services/location/index.js [hunk 1]
```diff
@@ -153,34 +174,54 @@ export async function getLocationSettings({ siteIds }) {
...
-      siteIds,
-    });
-
-    const uniqueIds = settings.map(setting => setting.id);
-
-    // The above API calls only return the ids. Make an additional
-    // call to getLocations so we can get additional details such
-    // as name, address, coordinates, etc.
-    if (uniqueIds.length) {
+    if (useV2) {
```
Comment (jbalboni) [inline src/applications/vaos/services/location/index.js:188]: The logic for V2 is flipped around. We have to call the facilities endpoint to get all the child facilities for the registered sites, then we pass those ids to the scheduling configurations endpoint to get the VATS settings.

File: src/applications/vaos/services/location/transformers.js [hunk 2]
```diff
@@ -43,9 +43,6 @@ export function transformDSFacilities(facilities) {
       requestSupported: facility.requestSupported,
       directSchedulingSupported: facility.directSchedulingSupported,
     },
-    managingOrganization: {
```
Comment (jbalboni) [inline src/applications/vaos/services/location/transformers.js:46]: This was not referenced in any active code

File: src/applications/vaos/services/location/transformers.v2.js [hunk 3]
```diff
@@ -32,7 +60,39 @@ export function transformParentFacilityV2(parentFacility) {
...
+
+  return allTypesOfCare.find(care => care.idV2 === id)?.id;
+}
+
+export function transformSettingsV2(settings) {
+  return settings.map(setting => ({
+    ...setting,
+    services: setting.services.map(service => ({
+      ...service,
+      id: getTypeOfCareIdFromV2(service.id),
```
Comment (jbalboni) [inline src/applications/vaos/services/location/transformers.v2.js:95]: We're converting the V2 ids to our old ids for now, will probably change this around in the future.

**Review by vbahinwillit — APPROVED** [review]
LGTM

**Review by LeKeve — APPROVED** [review]

**Pull request merged**